controllers/socket/socket.go

```diff
@@ -63,11 +64,36 @@ func SocketInit(so socketio.Socket) {
 			return string(bytes)
 		}
 
-		mapName, _ := js.Get("mapName").String()
-		lobbytype, _ := js.Get("type").String()
-		server, _ := js.Get("server").String()
-		rconPwd, _ := js.Get("rconpwd").String()
-		whitelist, _ := js.Get("whitelist").Int()
+		mapName, err := js.Get("mapName").String()
+		if err != nil {
```

Comment:
I think you can do all the checks and after that only do the `if err` block once.